prow/cluster/bazel-cache-configmap.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: bazel-cache-nginx-config
+  namespace: ${NAMESPACE}
+data:
+  default.conf: |
+    server {
+      listen       80;
+      server_name  localhost;
+
+      location /cache/ {
+          # The path to the directory where nginx should store the cache contents.
+          root /var/cache/bazel;
+          # Allow PUT
+          dav_methods PUT;
+          # Allow nginx to create the /ac and /cas subdirectories.
+          create_full_put_path on;
+          # The maximum size of a single file.
+          client_max_body_size 1G;
+          allow all;
```

Comment:
Should this be restricted (to the build machines, or their internal ip range), just to make sure? (For example "Environment variables leaking into an action" over at https://docs.bazel.build/versions/main/remote-caching.html made me wonder about this).

Reply:
I don't think this is an issue. This is an internal pod/service, nor reachable externally to the cluster.

Reply:
I see, resolving comment.